main.go

```diff
@@ -40,6 +42,21 @@ func init() {
 	}()
 }
 
+func RandomANString(len int) string {
```

Comment:
It's confusing to abbreviate names likes this, use e.g. "GenerateRandomName" or similar